### ISSUE TITLE
Is it worth to mention about "Polonious"?

### ISSUE THREAD

**Issue by vbauerster** [pretzelhammer/rust-blog#11]
I really enjoyed reading your `Lifetimes misconceptions` post, thank you! Recently I stumbled upon [Polonious](https://www.youtube.com/watch?v=_agDeiWek8w) talk given by Niko Matsakis. So I though it would be worth to mention about it as it's related to lifetime concept.

**Comment by pretzelhammer** [comment]
Thanks for your suggestion but Polonious, once it lands in stable, will have no visible effect on the language itself, the only difference will be that the Rust borrow checker's control flow analysis will be slightly better and it'll be able to verify the safety of slightly more complex corner cases. I don't think it will change any of the misconceptions in the article and I don't think it will change how users write Rust in any meaningfully noticeable way.

Also, I intentionally avoid discussing Rust RFCs, Rust internals, and Rust nightly experimental features in my blog because I want everything I pass along to my readers to be immediately useful, hence I only write about safe stable Rust. It's rarely worth it to mention Rust RFCs or nightly features because none of them have deadlines and there are RFCs and nightly features that have been around _for over 5 years_ and still haven't been finalized or merged into stable! Basically there's no guarantee of anything ever making it to stable within any reasonable timeframe, so it's not worth talking about given the scope of my blog.
